rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -170,10 +173,14 @@ impl FirmwareProcessor {
 
         txn.copy_request(slice)?;
 
-        if let Some(result) = ImageManifest::read_from(slice.as_bytes()) {
+        let opt = ImageManifest::read_from(slice.as_bytes());
+        let result = opt.is_some();
+        if cfi_launder(result) {
```

Comment:
I'm skeptical this is accomplishing much; we really want the code-gen to be executing the opt.is_some() line multiple times.

Reply:
I'll update this as part of the asymm. incr/decr proc. macro changes.